docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'
+
+Follow this document to publish a patch release of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix:
```

Comment:
Could add to the lead in line as the bullets start with the same phrase i.e. 

"Follow this document to publish a patch release of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:"